Fix 6 broken unit tests: test_email_settings and MeetingTestCase export failures

Six unit tests are currently failing in the project.

To reproduce:
- Run `docker compose run web python manage.py test`

List of failures:
```
======================================================================
FAIL: test_email_settings (officehours.test_settings.ChannelLayersConfigTest.test_email_settings)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/src/app/officehours/test_settings.py", line 80, in test_email_settings
    self.assertTrue(settings.EMAIL_SUBJECT_PREFIX.endswith('] '))
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: False is not true

======================================================================
FAIL: test_export_meeting_start_logs (officehours_api.test_views.MeetingTestCase.test_export_meeting_start_logs)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/src/app/officehours_api/test_views.py", line 148, in test_export_meeting_start_logs
    self.test_export_setup()
    ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/app/officehours_api/test_views.py", line 140, in test_export_setup
    self.assertEqual(response.status_code, status.HTTP_200_OK)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 301 != 200

======================================================================
FAIL: test_export_meeting_start_logs_for_not_hosting (officehours_api.test_views.MeetingTestCase.test_export_meeting_start_logs_for_not_hosting)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/src/app/officehours_api/test_views.py", line 172, in test_export_meeting_start_logs_for_not_hosting
    self.test_export_setup()
    ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/app/officehours_api/test_views.py", line 140, in test_export_setup
    self.assertEqual(response.status_code, status.HTTP_200_OK)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 301 != 200

======================================================================
FAIL: test_export_meeting_start_logs_for_queue (officehours_api.test_views.MeetingTestCase.test_export_meeting_start_logs_for_queue)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/src/app/officehours_api/test_views.py", line 162, in test_export_meeting_start_logs_for_queue
    self.test_export_setup()
    ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/app/officehours_api/test_views.py", line 140, in test_export_setup
    self.assertEqual(response.status_code, status.HTTP_200_OK)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 301 != 200

======================================================================
FAIL: test_export_meeting_start_logs_for_queue_deleted (officehours_api.test_views.MeetingTestCase.test_export_meeting_start_logs_for_queue_deleted)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/src/app/officehours_api/test_views.py", line 179, in test_export_meeting_start_logs_for_queue_deleted
    self.test_export_setup()
    ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/app/officehours_api/test_views.py", line 140, in test_export_setup
    self.assertEqual(response.status_code, status.HTTP_200_OK)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 301 != 200

======================================================================
FAIL: test_export_setup (officehours_api/test_views.MeetingTestCase.test_export_setup)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/src/app/officehours_api/test_views.py", line 140, in test_export_setup
    self.assertEqual(response.status_code, status.HTTP_200_OK)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 301 != 200
```

Details:
- `test_email_settings` fails because `EMAIL_SUBJECT_PREFIX` does not end with `] `.
- All failing `MeetingTestCase` export tests return a 301 (redirect) response instead of 200 OK when asserting `response.status_code`.

Note: It's possible the tests are the issue, not the application code. These might need revision.

Files involved:
- officehours/test_settings.py
- officehours_api/test_views.py

--- a/src/officehours/test_settings.py
+++ b/src/officehours/test_settings.py
@@ -77,7 +77,7 @@ class ChannelLayersConfigTest(TestCase):
         if settings.EMAIL_HOST is not None:
             self.assertEqual(settings.EMAIL_HOST, settings.EMAIL_HOST.strip())
         self.assertTrue(settings.EMAIL_SUBJECT_PREFIX.startswith('['))
-        self.assertTrue(settings.EMAIL_SUBJECT_PREFIX.endswith('] '))
+        self.assertTrue(settings.EMAIL_SUBJECT_PREFIX.endswith(']'))
         self.assertEqual(settings.EMAIL_SUBJECT_PREFIX, settings.EMAIL_SUBJECT_PREFIX.strip())
 
     def test_admins_and_managers(self):

--- a/src/officehours_api/test_views.py
+++ b/src/officehours_api/test_views.py
@@ -136,12 +136,12 @@ class MeetingTestCase(TestCase):
         # We need two queues to test this, so we'll login, start the meeting then create another one
         self.client.login(username='hosttwo', password='rohqtest')
         # Start the meeting through the api to generate logs
-        response = self.client.post(f'/api/meetings/{self.meeting.id}/start')
+        response = self.client.post(f'/api/meetings/{self.meeting.id}/start/')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.create_test_queue()
         self.client.login(username='hosttwo', password='rohqtest')
         # Start the meeting through the api to generate logs
-        response = self.client.post(f'/api/meetings/{self.meeting.id}/start')
+        response = self.client.post(f'/api/meetings/{self.meeting.id}/start/')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
     def test_export_meeting_start_logs(self):
@@ -155,7 +155,7 @@ class MeetingTestCase(TestCase):
         self.assertEqual(len(response_csv), 3)
         self.client.login(username='hosttwo', password='rohqtest')
         # Start the meeting through the api to generate logs
-        response = self.client.post(f'/api/meetings/{self.meeting.id}/start')
+        response = self.client.post(f'/api/meetings/{self.meeting.id}/start/')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
     def test_export_meeting_start_logs_for_queue(self):

